CHANGELOG.md

```diff
@@ -6,13 +6,17 @@ All notable changes to Wallaroo will be documented in this file.
 
 ### Fixed
 
+- Argument parsers in machida no longer require a program name to be passed
```

Comment:
This description seems to imply that we changed existing behavior but not that the previous behavior was broken. Maybe I'm misunderstanding what was broken previously.

Seems like more of a `changed` item than `fixed` item if I'm understanding it correctly.

Reply:
It causes Python `sys._argv` to fail at runtime breaking a chunk of the Python standard library. I'd consider it a bug rather than a change.